packages/wdio-cli/src/commands/run.ts

```diff
@@ -146,23 +146,22 @@ export async function launch(wdioConfPath: string, params: Partial<RunCommandArg
 }
 
 export async function handler(argv: RunCommandArguments) {
-    const { configPath, ...params } = argv
+    const { configPath = 'wdio.conf.js', ...params } = argv
+    const configPathNoExtension = configPath.substring(0, configPath.lastIndexOf('.'))
 
-    const canAccessConfigPath = await fs.access(configPath).then(
+    const canAccessConfigPath = await fs.access(`${configPathNoExtension}.js`).then(
         () => true,
-        () => false
+        () => fs.access(`${configPathNoExtension}.ts`).then(
+            () => true, () => false
+        )
     )
+    const wdioConf = configPath.includes(process.cwd())
+        ? configPath : path.join(process.cwd(), configPath)
```

Comment:
```suggestion
        ? configPath
        : path.join(process.cwd(), configPath)
```

Reply:
This wasn't formatted for me automatically and ESLint doesn't have a problem with it.
Are the EditorConfig rules supposed to do that?

Reply:
There is EsLint rule for that, I can work on adding one to make this more obvious.